cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -1675,62 +1573,6 @@ private KafkaVersionChange getKafkaVersionChange(StatefulSet kafkaSts) {
             return withVoid(blockingPromise.future());
         }
 
-        Future<ReconciliationState> kafkaNodePortExternalListenerStatus() {
```

Comment:
was it removed by mistake or on purpose? isn't it something added recently by Jakub?

Reply:
This is probably some rebasing incident :( let me fix this.